src/options/empty-attributes.ts

```diff
@@ -0,0 +1,80 @@
+import { AttributeToken } from 'pug-lexer';
+import { ChoiceSupportOption, PathArraySupportOption } from 'prettier';
+import { CATEGORY_PUG } from '.';
+
+// eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
You now can remove all these `disable` statements
We have updated prettier types